include/deal.II/numerics/data_postprocessor.h

```diff
@@ -591,8 +595,169 @@ class DataPostprocessorScalar : public DataPostprocessor<dim>
+ * different directions. But that is because the solution is only continuous:
+ * in general, the gradient is discontinuous across edges, and so the multiple
+ * vectors originating from each vertex simply represent the differing
+ * gradients of the solution at each adjacent cell.
+ *
+ * The output above -- namely, the gradient $\nabla u$ of the solution --
+ * corresponds to the "strain" in linear elasticity if the solution were scalar.
+ * It is very small in the central part of the domain because in step-6 we are
+ * solving an equation that has a coefficient $a(\mathbf x)$ that is large in
+ * the central part and small on the outside. On the other hand, the "stress"
+ * corresponds to the quantity $a(\mathbf x) \nabla u(\mathbf x)$. For the
+ * solution of that equation, the stress should be continuous and about as large
+ * in the interior part of the domain as on the part closer to the boundary.
+ * This is easily verified by the following modification of the postprocessor:
+ * @code
+ * template <int dim>
+ * class StressPostprocessor : public DataPostprocessorVector<dim>
+ * {
+ * public:
+ *   StressPostprocessor ()
+ *     :
+ *     // like above, but now also make sure that DataOut provides
+ *     // us with coordinates of the evaluation points:
+ *     DataPostprocessorVector<dim> ("stress",
+ *                                   update_gradients | update_quadrature_points)
+ *   {}
+ *
+ *   virtual
+ *   void
+ *   evaluate_scalar_field (const DataPostprocessorInputs::Scalar<dim> &input_data,
+ *                          std::vector<Vector<double> >               &computed_quantities) const
+ *   {
+ *     AssertDimension (input_data.solution_gradients.size(),
+ *                      computed_quantities.size());
+ *
+ *     for (unsigned int p=0; p<input_data.solution_gradients.size(); ++p)
+ *       {
+ *         AssertDimension (computed_quantities[p].size(), dim);
+ *         for (unsigned int d=0; d<dim; ++d)
+ *           // like above, but also multiply the gradients with
+ *           // the coefficient evaluated at the current point:
+ *           computed_quantities[p][d]
+ *             = input_data.solution_gradients[p][d];
+ *       }
+ *   }
+ * };
+ * @endcode
+ * With this postprocessor, we get the following picture of the "stress"
+ * variable:
+ *
+ * @image html data_postprocessor_vector_2.png
+ *
+ * As the background color shows, the gradient times the coefficient is now
+ * a continuous function. There are (large) vectors around the interface
+ * where the coefficient jumps (at half the distance between the center of the
+ * disk to the perimeter) that seem to point in the wrong direction; this is
+ * an artifact of the fact that the solution has a discontinuous gradient
+ * at these points and that the numerical solution on the current grid
+ * does not adequately resolve this interface. This, however, is not
+ * important to the current discussion.
+ *
+ *
+ * <h3> Extension to the real "stress" </h3>
```

Comment:
If we do change "stress" to "heat flux" in the above discussion, then I think we could just change this title to something like "Extension to tensor valued fields" and keep what you've got written below. The only thing we'd need to change is "real stress" to "stress".